File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/AddressDto.java
package com.sudo248.orderservice.controller.order.dto;

import com.sudo248.orderservice.repository.entity.order.Location;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AddressDto {
    private String addressId;

    private int provinceID;

    private int districtID;

    private String wardCode;

    private String provinceName;

    private String districtName;

    private String wardName;

    private String address;

    private String fullAddress;
}

File: server/product-service/src/main/kotlin/com/sudoo/productservice/dto/AddressDto.kt
package com.sudoo.productservice.dto

data class AddressDto(
    val addressId: String? = null,
    val provinceID: Int,
    val districtID: Int,
    val wardCode: String,
    val provinceName: String,
    val districtName: String,
    val wardName: String,
    val address: String,
    val location: Location? = null,
    val fullAddress: String? = null,
)
